docs/ref/item.rst

```diff
@@ -26,6 +26,10 @@ Say for instance you have 2 parameters configured: Adults (id: adults), and Chil
 
 Booking parameters have many options, and can be configured to control inventory in very specific ways.  See the Checkfront support centre or ask us for more information.
 
+Public API
+----------
+If the Public API is enabled, non-authenticated users can make requests to the */item*, */item/{id}*, */item/cal*, and */item/{id}/cal* end points. The responses between the internal and public API versions are identical for the *item/* and *item/{id}* endpoints. For the *item/cal* and *item/{id}/cal* endpoints the response will return 1 if the item is available instead of the available inventory. This is the same functionality as a customer viewing the item or availability calendar on the `Hosted Booking Page <https://support.checkfront.com/hc/en-us/articles/115004917593-Hosted-Booking-Page>`_.
```

Comment:
Might be worth mentioning that while the response structure is identical, public requests will use customer rules and item visibility instead of staff rules and item visibility.

Reply:
Resolved in latest commit.